hloc/localize_inloc.py

```diff
@@ -140,7 +144,10 @@ def main(dataset_dir, retrieval, features, matches, results, skip_matches=None):
             dataset_dir, q, db, feature_file, match_file, skip_matches
         )
 
-        poses[q] = (ret["qvec"], ret["tvec"])
+        poses[q] = (
```

Comment:
Could we instead use `poses[q] = ret["cam_from_world"]`, define a new function
```
write_poses(poses: Mapping[str, pycolmap.Rigid3d], path: str, prepend_camera_name: bool)
```
 in `utils/io.py`, and use in both localize_sfm.py and localize_inloc.py?

Reply:
Very nice idea!